pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -634,6 +634,7 @@ public static class Offline {
     public static class AssignmentStrategy {
       public static final String BALANCE_NUM_SEGMENT_ASSIGNMENT_STRATEGY = "BalanceNumSegmentAssignmentStrategy";
       public static final String REPLICA_GROUP_SEGMENT_ASSIGNMENT_STRATEGY = "ReplicaGroupSegmentAssignmentStrategy";
+      public static final String ALL_SERVERS_SEGMENT_ASSIGNMENT_STRATEGY = "AllServersSegmentAssignmentStrategy";
```

Comment:
This needs to be changed to dim table based.

Reply:
updated